app/views/books/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<% @books.each do |book|%>
```

Comment:
Perfect, looks like all we need for or user view for the time being. I like the fact that its creating a separate div for each book, should make styling uniform and clean.